swmm-toolkit/src/swmm/toolkit/solver_enum.py

```diff
@@ -0,0 +1,284 @@
+#
```

Comment:
I suggest combining contents of output_enum.py and solver_enum.py into a single file named toolkit_enum.py